activerecord/lib/active_record/railtie.rb

```diff
@@ -141,6 +141,11 @@ class Railtie < Rails::Railtie # :nodoc:
             cache = ActiveRecord::ConnectionAdapters::SchemaCache.load_from(filename)
             next if cache.nil?
 
+            if cache.postgresql_adapter?
```

Comment:
Same comment as above. Can we implement with a more general interface (ex: `adapter.update_from_schema_cache(cache)`) even if for now it's a no-op for everything but postgres? That will maintain a general interface in the railtie and co-locate adapter-specific logic.